docarray/array/mixins/evaluation.py

```diff
@@ -43,14 +48,16 @@ class EvaluationMixin:
     @_evaluate_deprecation
     def evaluate(
         self,
-        metric: Union[str, Callable[..., float]],
+        metrics: Union[
+            Union[str, Callable[..., float]], List[Union[str, Callable[..., float]]]
```

Comment:
Do the same deprecation with `metric` and `metric_name`. Make sure the name is equivalent, so that we do not add breaking changes

Reply:
I agree, there is not so much we can do

Reply:
I agree as well on returning always a dict otherwise it can be confusing even though it is a breaking change